service/server_salt.go

```diff
@@ -53,8 +48,8 @@ func (randomSaltGenerator) IsServerSalt(salt []byte) bool {
 	return false
 }
 
-// RandomSaltGenerator is a basic SaltGenerator.
-var RandomSaltGenerator ServerSaltGenerator = randomSaltGenerator{}
+// RandomServerSaltGenerator is a basic ServerSaltGenerator.
+var RandomServerSaltGenerator ServerSaltGenerator = randomSaltGenerator{}
```

Comment:
Should we rename this to `randomServerSaltGenerator{}`?  It's a little confusing to have different constants in the `service` and `shadowsocks` packages with the same name.

Reply:
Done